add_kubernetes.md

```diff
@@ -141,82 +141,14 @@ To configure the Kubernetes Metrics Collector to use a Wavefront proxy that's al
     
 #### Install and Configure the Collector on OpenShift Enterprise 3.x
 
-**Note**: The Helm or manually-installed Kubernetes Metrics Collector and Wavefront proxy is deprecated. Our new Observability for Kubernetes Operator replaces the Helm or manually installed Kubernetes Metrics Collector and Wavefront proxy for all Kubernetes Distributions except for OpenShift Container Platform. For more information, see [Obsolescence and Remediation](https://docs.wavefront.com/wavefront_obsolescence_policy.html#kubernetes-integration).
+**Note**: The Helm or manually installed Kubernetes Metrics Collector and Wavefront proxy is deprecated and has reached EOL. Our Observability for Kubernetes Operator replaces the Helm or manually installed Kubernetes Metrics Collector and Wavefront proxy for all Kubernetes Distributions except for OpenShift Container Platform.
```

Comment:
Our Observability for Kubernetes Operator replaces the Helm or manually installed Kubernetes Metrics Collector and Wavefront proxy for all Kubernetes Distributions except for the OpenShift Container Platform.